### DISCUSSION TITLE
Asking Siri to turn on lights in kitchen starts the dishwasher

### DISCUSSION THREAD

**Discussion by fitur** [thoukydides/homebridge-homeconnect#176] [Q&A]
This is a rather ridiculous issue. I've placed the dishwasher (via Homebridge-Homeconnect) in my kitchen room in the Home app.
When I ask Siri to turn on the lights in the kitchen, the dishwasher default program starts. Any way to get it to not start without setting programs to select only in the Homebridge configuration?

**Comment by thoukydides** [comment]
It doesn’t do that for me. In my kitchen HomeKit has this plugin with switches to start oven programs, and (Philips Hue) lights. Asking Siri to “turn on the kitchen lights” only affects the lights, not the oven programs.

Are you sure this isn’t a case of Siri mishearing the command? A command like “turn on the kitchen” would turn on both lights and switches, which would then attempt to power-on and start programs controlled by this plugin…
